### ISSUE TITLE
SNOW-1613751: snowflake-export-cert tool is broken

### ISSUE THREAD

**Issue by Baguage** [snowflakedb/snowflake-connector-python#2015]
### Python version

Python 3.12.4

### Operating system and processor architecture

Linux-5.4.0-153-generic-x86_64-with-glibc2.38

### Installed packages

```python
asn1crypto==1.5.1
certifi==2024.7.4
cffi==1.16.0
charset-normalizer==3.3.2
cryptography==42.0.8
filelock==3.15.4
idna==3.7
packaging==24.1
platformdirs==4.2.2
pycparser==2.22
PyJWT==2.9.0
pyOpenSSL==24.2.1
pytz==2024.1
requests==2.32.3
snowflake-connector-python==3.12.0
sortedcontainers==2.4.0
tomlkit==0.13.0
typing_extensions==4.12.2
urllib3==2.2.2
```

### What did you do?

```python
snowflake-connector-python comes with `snowflake-export-certs` tool, but it appears to be broken:


(venv) root@barbarian:~/venv# bin/snowflake-export-certs
Traceback (most recent call last):
  File "/root/venv/bin/snowflake-export-certs", line 5, in <module>
    from snowflake.connector.tool.export_certs import main
ModuleNotFoundError: No module named 'snowflake.connector.tool.export_certs'
```

I believe it can be fixed by removing https://github.com/snowflakedb/snowflake-connector-python/blob/main/setup.cfg#L79 (`snowflake-export-certs = snowflake.connector.tool.export_certs:main`) - there is no `snowflake.connector.tool.export_certs` module as far as I can tell.
```


### What did you expect to see?

No error or no `snowflake-export-certs` command in the package

### Can you set logging to DEBUG and collect the logs?

```bash
import logging
import os

for logger_name in ('snowflake.connector',):
    logger = logging.getLogger(logger_name)
    logger.setLevel(logging.DEBUG)
    ch = logging.StreamHandler()
    ch.setLevel(logging.DEBUG)
    ch.setFormatter(logging.Formatter('%(asctime)s - %(threadName)s %(filename)s:%(lineno)d - %(funcName)s() - %(levelname)s - %(message)s'))
    logger.addHandler(ch)
```

**Comment by sfc-gh-dszmolka** [comment]
hi - thank you for pointing this out. Removing reference in https://github.com/snowflakedb/snowflake-connector-python/pull/2023 because module itself has been removed/consolidated 6 years ago.

**Comment by sfc-gh-dszmolka** [comment]
PR merged and will be part of the next release

**Comment by sfc-gh-aling** [comment]
we have released the connector v3.12.1 today with the fix, thanks!
